pkg/operation/botanist/health_check.go

```diff
@@ -600,12 +600,31 @@ func (b *Botanist) checkSystemComponents(
 	if exitCondition := checker.CheckExtensionCondition(condition, extensionConditions); exitCondition != nil {
 		return exitCondition, nil
 	}
-	if established, err := b.CheckVPNConnection(ctx, logrus.NewEntry(logger.NewNopLogger())); err != nil || !established {
-		msg := "VPN connection has not been established"
+
+	podsList := &corev1.PodList{}
+	if err := b.K8sShootClient.Client().List(ctx, podsList, client.InNamespace(metav1.NamespaceSystem), client.MatchingLabels{"type": "tunnel"}); err != nil {
+		return nil, err
+	}
+
+	if len(podsList.Items) == 0 {
+		return nil, fmt.Errorf("no tunnels are currently deployed to perform health-check on")
```

Comment:
I think that returning an error here will lead to `Unknown` status. Probably it's more accurate to report `False`, i.e, construct a condition with `b.FailedCondition()`?